core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -0,0 +1,134 @@
+<script type="text/ng-template" id="components/explorationFooter">
+  <div class="footer navbar-fixed-bottom oppia-exploration-footer">
+    <div class="row">
+      <div class="col-sm-6">
+        <ul class="author-profile">
+          <li>
+            <img ng-src="<[getStaticImageUrl('/general/apple.svg')]>" class="exploration-footer-img">
+          </li>
+          <li class="hover-link"
+              ng-mouseenter="hovering=true"
+              ng-mouseleave="hovering=false">
+            <div class="dropup" ng-class="{'open':hovering}">
+              <a id="dropdownMenu2" class="dropdown-toggle" ng-class="{'hovered-text':hovering}" data-toggle="dropdown"
+                 ng-if="contributorNames.length > 0">
+                <h4 translate="I18N_FOOTER_AUTHOR_PROFILE" class="author-profile-text"></h4>
+              </a>
+              <ul class="author-profile-dropdown-menu dropdown-menu" aria-labelledby="dropdownMenu2" role="menu">
+                <li ng-repeat="name in contributorNames | limitTo: 5" style=>
```

Comment:
Yep, I believe so. If you click on the link, you should visit /profile/{{name}} (assuming this is in the context of "name in contributorNames").